cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,24 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('User', () => {
-  before(() => {
+import UserPageObject from '../support/pages/user.pageObject';
+
+const userPage = new UserPageObject();
 
+describe('User', () => {
+  let user;
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+    cy.register();
   });
 
   it.skip('should be able to follow the another user', () => {
```

Comment:
It's a best practice not to leave skipped tests. If the test is not ready yet, consider adding a TODO comment to remind you or other developers to implement it later.